tests/static_assert_test.c

```diff
@@ -0,0 +1,36 @@
+// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+#include "macro_utils/macro_utils.h"
+#include <string.h>
```

Comment:
The C standard headers always come before our other headers.

Order should be:

- starndard headers
- OS headers (windows, etc.)
- our other headers

Reply:
done